eng/common/scripts/Helpers/DevOps-WorkItem-Helpers.ps1

```diff
@@ -884,6 +885,25 @@ function UpdatePackageVersions($pkgWorkItem, $plannedVersions, $shippedVersions)
   "value": "$shippedPackages"
 }
 "@
+
+    # If we shipped a version after we set "In Release" state then reset the state to "Next Release Unknown"
+    if ($pkgWorkItem.fields["System.State"] -eq "In Release")
+    {
+      $lastShippedDate = [DateTime]$newShippedVersions[0].Date
+      $markedInReleaseDate = ([DateTime]$pkgWorkItem.fields["Microsoft.VSTS.Common.StateChangeDate"])
+
+      # We just shipped so lets set the state to "Next Release Unknown"
+      if ($markedInReleaseDate -le $lastShippedDate)
+      {
+        $fieldUpdates += @'
+{
+"op": "replace",
+"path": "/fields/State",
+"value": "Next Release Unknown"
```

Comment:
Indentation?

Reply:
Intentional because it is a verbatim string.

Reply:
Oh but I guess there could still be some indention on the json elements.